examples/testdata/wasi_arg.wat

```diff
@@ -0,0 +1,61 @@
+;; $wasi_arg is a WASI command which copies null-terminated args to stdout.
+(module $wasi_arg
```

Comment:
added a %.wat program (it works in other runtimes too). This is intentionally a blackbox test (not sneaking looks at internals to assume things work.